etc/profile-m-z/nextcloud-desktop.profile

```diff
@@ -0,0 +1,48 @@
+# Firejail profile for nextcloud-desktop
+# Description: Nextcloud desktop client
+# This file is overwritten after every install/update
+# Persistent local customizations
+include nextcloud-desktop.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${DOCUMENTS}
+noblacklist ${HOME}/Nextcloud
+noblacklist ${HOME}/.config/Nextcloud
+noblacklist ${HOME}/.local/share/Nextcloud
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+
+mkdir ${HOME}/Nextcloud
+mkdir ${HOME}/.config/Nextcloud
+mkdir ${HOME}/.local/share/Nextcloud
+whitelist ${DOCUMENTS}
+whitelist ${HOME}/Nextcloud
+whitelist ${HOME}/.config/Nextcloud
+whitelist ${HOME}/.local/share/Nextcloud
+include whitelist-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+netfilter
+no3d
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+
+private-dev
+private-tmp
+
```

Comment:
Does it need full D-Bus access?

Reply:
I don't think so.
What do you suggest?

Reply:
If it works `dbus-{user,system} none`.

Reply:
I was able to use only `dbus-system none`
`dbus-user filter` and `dbus-user filter` didn't allow login

Reply:
The flatpak permissions look like:
```
[Session Bus Policy]
com.canonical.AppMenu.Registrar=talk
org.kde.StatusNotifierWatcher=talk
org.freedesktop.Notifications=talk
org.kde.*=own
org.freedesktop.secrets=talk
```
so you could try
```
dbus-user filter
# Maybe not necessary 
dbus-user.own com.nextcloud.desktopclient.nextcloud
# If it is required, we will add it
dbus-user.talk org.freedesktop.secrets
```

Reply:
Works perfectly!
Now we can even use `dbus-user filter`
`dbus-user.talk org.freedesktop.secrets` is useful in case of meeting calls.
Resolved! :-)

```
dbus-user filter
dbus-user.own com.nextcloud.desktopclient.nextcloud
dbus-user.talk org.freedesktop.secrets
dbus-system none
```